cmd/ctr/events.go

```diff
@@ -57,13 +57,11 @@ var eventsCommand = cli.Command{
 
 func getEventOutput(env *eventsapi.Envelope) (string, error) {
 	out := ""
-
-	var de events.DynamicEvent
-	if err := events.UnmarshalEvent(env.Event, &de); err != nil {
+	v, err := typeurl.UnmarshalAny(env.Event)
```

Comment:
I went back and forth on dynamic vs allocated `UnmarshalAny` and in certain cases, the dynamic approach avoids a serialization if you use the `Is` function.